Source/Scene/Expression.js

```diff
@@ -1404,6 +1413,14 @@ define([
         return expressions;
     }
 
+    function fract(number) {
```

Comment:
Place these functions closer to the top, maybe right before `var binaryFunctions = {`.